app/javascript/components/mentoring/queue/SolutionList.jsx

```diff
@@ -12,32 +14,63 @@ export function SolutionList({ request, setPage }) {
     resolvedData,
     latestData,
   } = usePaginatedRequestQuery('mentor-solutions-list', request)
+  const [tooltipTriggerElement, setTooltipTriggerElement] = useState(null)
+  const [tooltipElement, setTooltipElement] = useState(null)
+  const [tooltipEndpoint, setTooltipEndpoint] = useState(null)
+  const { styles } = usePopper(tooltipTriggerElement, tooltipElement)
+
+  const showTooltip = (referenceElement, tooltipUrl) => {
+    setTooltipTriggerElement(referenceElement)
+    setTooltipEndpoint(tooltipUrl)
+  }
+
+  const hideTooltip = () => {
+    setTooltipEndpoint(null)
+    setTooltipTriggerElement(null)
+  }
 
   return (
     <div>
       {isLoading && <Loading />}
       {isError && <p>Something went wrong</p>}
       {isSuccess && (
-        <table>
-          <thead>
-            <tr>
-              <th>Track icon</th>
-              <th>Mentee avatar</th>
-              <th>Mentee handle</th>
-              <th>Exercise title</th>
-              <th>Starred?</th>
-              <th>Mentored previously?</th>
-              <th>Status</th>
-              <th>Updated at</th>
-              <th>URL</th>
-            </tr>
-          </thead>
-          <tbody>
-            {resolvedData.results.map((solution, key) => (
-              <Solution key={key} {...solution} />
-            ))}
-          </tbody>
-        </table>
+        <>
+          <table>
+            <thead>
+              <tr>
+                <th>Track icon</th>
+                <th>Mentee avatar</th>
+                <th>Mentee handle</th>
+                <th>Exercise title</th>
+                <th>Starred?</th>
+                <th>Mentored previously?</th>
+                <th>Status</th>
+                <th>Updated at</th>
+                <th>URL</th>
+              </tr>
+            </thead>
+            <tbody>
+              {resolvedData.results.map((solution, key) => (
+                <Solution
+                  key={key}
+                  {...solution}
+                  onMouseEnter={(e) =>
+                    showTooltip(e.target, solution.tooltipUrl)
+                  }
+                  onMouseLeave={() => hideTooltip()}
+                />
+              ))}
+            </tbody>
+          </table>
+          <div ref={setTooltipElement}>
+            {tooltipEndpoint && (
+              <Tooltips.Student
+                endpoint={tooltipEndpoint}
+                styles={styles.popper}
+              />
+            )}
```

Comment:
This will break when it's falsy but not `null` -- `false` works on regular React but not all other renderers.

```suggestion
            {tooltipEndpoint ? (
              <Tooltips.Student
                endpoint={tooltipEndpoint}
                styles={styles.popper}
              />
            ) : null}
```